tools/build-and-push-static-serving.sh

```diff
@@ -11,6 +12,17 @@ if [ "$DONT_BUILD" != "true" ]; then
 fi
 
 if [ "$DONT_PUSH" != "true" ]; then
+  BUCKET=$1
+  if ! [[ $BUCKET == who-* ]]; then
+    echo "Required argument: project id starting with 'who-' : $PROJECT"
+    exit 1
+  fi
+  # TODO: remove once .github/workflows/static-content.yaml is updated
+  if [[ $BUCKET == *static-content* ]]; then
+    echo "Deprecated argument for bucket name: $BUCKET"
+  else
+    BUCKET=$BUCKET-static-content
```

Comment:
i assume this whole script will go away, but it should be possible to get the result from Terraform. Which sadly is quite difficult.